src/operator/src/insert.rs

```diff
@@ -310,14 +302,26 @@ impl Inserter {
             .into_iter()
             .filter_map(|(k, v)| v.map(|v| (k, v)))
         {
-            for flownode in peers {
+            if peers.len() == 1 {
+                // fast path, zero copy
                 inserts
-                    .entry(flownode.clone())
+                    .entry(peers[0].clone())
                     .or_default()
                     .requests
-                    .extend(reqs.requests.clone());
+                    .extend(reqs.requests);
+                continue;
+            } else {
+                // TODO(discord9): need to split requests to multiple flownodes
```

Comment:
cc @discord9

Reply:
make sense since we are only at standalone mode now